### PR TITLE
Vue3 apollo quasar/feat/language filter functionality

### LINKED ISSUE #391: [vue3-apollo-quasar] Implement language filter functionality
## Background

In another ticket, a component was created with the language dropdown button. The purpose of this ticket is to add the functionality and populate the fields for that button. The options that should be available to the user are listed below - when a user selects an option, the repository list on the page should be filtered to only include ones that match the user’s selection.

## Acceptance

- [ ] when an option is selected, will filter the current results based on the selected option
- [ ] language options should show all (default) and list of languages from users’ repos
- [ ] only one selection allowed, noted with a check mark icon

### PR DESCRIPTION
**Pull request by hdJerry**
Resolves #391 

- [x] when an option is selected, will filter the current results based on the selected option
- [x] language options should show all (default) and list of languages from users’ repos
- [x] only one selection allowed, noted with a check mark icon

_used a Reactive variable to store the selected language_

### CODE REVIEW

File: vue3-apollo-quasar/src/init/setupGraphQL.ts [hunk 1]
```diff
@@ -25,7 +26,20 @@ const authLink = setContext((_, { headers }) => {
...
 // Cache implementation
-const cache = new InMemoryCache();
+const cache = new InMemoryCache({
+  typePolicies: {
+    Query: {
+      fields: {
+        language: {
+          //The name we will be querying
+          read() {
+            return filteredLanguage(); //Returns the updated value of counts
```
[the remark this answers is not in the document]
Comment (hdJerry) [reply vue3-apollo-quasar/src/init/setupGraphQL.ts:36]: its not a method. its Reactive variable. so its just a variable

File: vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue [hunk 2]
```diff
@@ -27,7 +26,7 @@
 
             <q-tab-panel name="repositories">
               <q-list v-if="repos" separator>
-                <q-item v-for="repo in repos" :key="repo.id">
+                <q-item v-for="repo in filterRepoDataByLanguage" :key="repo.id">
```
[the remark this answers is not in the document]
Comment (hdJerry) [reply vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue:29]: yes. for now, until we bring all the filters together.

File: vue3-apollo-quasar/src/styles/quasar.variables.scss [hunk 3]
```diff
@@ -6,6 +6,7 @@ $primary-100: #f6f8fa;
 $secondary: #586069;
 $secondary-100: #d0d7de;
 $secondary-200: #57606a;
+$secondary-300: #eaecef;
```
[the remark this answers is not in the document]
Comment (hdJerry) [reply vue3-apollo-quasar/src/styles/quasar.variables.scss:9]: I get you champ Thanks. we could raise another ticket for this. You get? so we don't go beyond or away even though its a small fix.

File: vue3-apollo-quasar/src/init/setupGraphQL.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (AllanJeremy) [reply vue3-apollo-quasar/src/init/setupGraphQL.ts:36]: got it

File: vue3-apollo-quasar/src/styles/quasar.variables.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (AllanJeremy) [reply vue3-apollo-quasar/src/styles/quasar.variables.scss:9]: Sure, kindly create one

File: vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (AllanJeremy) [reply vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue:29]: Understood, I'm not suggesting any functional changes, simply a rename of the existing variable

**Review by AllanJeremy — APPROVED** [review]
:shipit:

**Pull request merged**